modules/packages/DistributedDeque.chpl

```diff
@@ -84,24 +84,24 @@
 
     var deque = new DistDeque(int, cap=maxElem, targetLocales=ourLocales);
 
-  The deque can be used as a queue by using the :proc:`enqueue` and :proc:`dequeue` convenience
+  The deque can be used as a queue by using the :proc:DistDeque.`enqueue` and :proc:`DistDeque.dequeue` convenience
```

Comment:
Something is funny with this one. The first DistQueue is not in quotes, and I kinda expected a ~ before them?

Reply:
My mistake. Fixed in latest changes.